images/common/utils.sh

```
@@ -236,18 +236,25 @@ function openvpn_config_checksum {
+	if [ -n "$CONF_FILE" ]; then
+		mv -f -- "$CONF_FILE" openvpn.conf
+	elif [ ! -f openvpn.conf ]; then
+		echo "ERROR: no OpenVPN config file found after extraction" >&2
+		return 1
+	fi
 }
 
 function crl_download {
 	curl --silent --insecure --output revoked.crl \
-	${DASHBOARD_INTERNAL}/admin/pki/ca/x509/ca/${CA_UUID}.crl
+		${DASHBOARD_INTERNAL}/admin/pki/ca/x509/ca/${CA_UUID}.crl
```

Shellcheck (0.11.0)

[info] Double quote to prevent globbing and word splitting.

(SC2086)